match-lms-release/README.md

```diff
@@ -51,6 +53,50 @@ The release step will fail to write to `package.json` if you have branch protect
 [Learn how to set up the D2L_GITHUB_TOKEN...](https://github.com/BrightspaceUI/actions/blob/main/docs/branch-protection.md)
 
 ## Release Increments
+
 The first commit made during any given LMS release will update the Major and/or Minor versions to match the LMS version provided. Other commits during the same LMS release will trigger patch releases.
 
-Pull requests merged into maintenance branches created by the action (ex: `release/2022.2.x`) or named for a minor version (ex: `1.7.x`) will trigger patch releases for the maintenance branch version.
+Pull requests merged into maintenance branches created by the action (ex: `release/2022.2.x`) or named for a minor version (ex: `1.7.x`) or major version (ex: `1.x`) will trigger patch releases for the maintenance branch version.
+
+## NPM Package Deployment
+
+If you'd like the action to deploy your package to NPM, set the `NPM` option to `true`.
+
+NPM deployments for maintenance branches (ex: `release/2022.2.x`, `1.7.x`, or `1.x`) will be annotated with a tag corresponding to the branch version (ex: `release-2022.2.x`, `release-1.7.x`, or `release-1.x`). All other deployments will use NPM's default tag of `latest`.
+
+### CodeArtifact
+
+To publish to CodeArtifact, ensure that prior to running the `incremental-release` step that the [add-registry](https://github.com/Brightspace/codeartifact-actions/tree/main/npm) and the [get-authorization-token](https://github.com/Brightspace/codeartifact-actions/tree/main/get-authorization-token) steps have been run.
+
+### NPM
+
+Setup Node with the `registry-url` option:
+
+```yml
+- name: Setup Node
+  uses: Brightspace/third-party-actions@actions/setup-node
+    with:
+     registry-url: 'https://registry.npmjs.org'
+```
```

Comment:
The npm default registry should be setup by default so the `registry-url:` parts of this example are probably not needed.

Reply:
[This doc](https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages) shows them, but yeah if they're not needed then that's awesome.

Reply:
Yeah core doesn't specify them and it works. Maybe that's handled by semantic release but I don't think so
https://github.com/BrightspaceUI/core/blob/main/.github/workflows/release.yml#L17-L20